GoogleForCreators/web-stories-wp
# PPC 2.0: Panels for the cards

## Context

PPC 2.0. [Designs for this behavior](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=8393%3A38)
Notes from design:
1. Each expanded categories must have a fixed height of 560px which is scrollable.
2. The divider between each categories should be aligned down with the height of the expandex box.

## Summary

Adding new panels for the cards in the new Pre Publish Checklist.

## Relevant Technical Choices

We decided to go with a new, lighter weight panel for this section because:
- The current [`SimplePanel`](https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/components/panels/panel/simplePanel.js) (used in the inspector) tries to solve too many problems at once, and we can make something lightweight that is also more accessible since it should be a tab panel (one section should only be open at a time).
- These panels don't need to be draggable and the size doesn't need to change

## To-do

n/a

## User-facing changes

No user facing changes. All work done in storybook:

<img src="https://user-images.githubusercontent.com/22185279/123163869-78319980-d42f-11eb-818c-3cc29f2b9e11.gif" width="300px" />

## Testing Instructions

Storybook -> Stories Editor -> Tablist. Make sure to scroll or close the actions tab to see everything.

Should be accessible with keyboard and assistive technologies like voiceover or screenreaders.

### QA






## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7918
